org.gmusicbrowser.gmusicbrowser.json

```diff
@@ -0,0 +1,37 @@
+{
+    "app-id": "org.gmusicbrowser.gmusicbrowser",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "gmusicbrowser",
+    "finish-args": [
+        "--share=ipc", 
+        "--socket=x11",
+        "--socket=wayland",
+        "--share=network",
+        "--filesystem=xdg-music",
+        "--socket=pulseaudio",
+        "--socket=session-bus",
```

Comment:
Does it need access to the whole session-bus?

Reply:
Hmm, probably not.

Reply:
I removed it for now, Ill nail down which specific DBUS channel it needs later